vnext/Microsoft.ReactNative/ReactHost/ReactInstanceWin.h

```diff
@@ -41,16 +41,21 @@ class ReactContext final : public Mso::UnknownObject<IReactContext> {
  public:
   ReactContext(
       Mso::WeakPtr<ReactInstanceWin> &&reactInstance,
-      winrt::Microsoft::ReactNative::IReactPropertyBag const &properties) noexcept;
+      winrt::Microsoft::ReactNative::IReactPropertyBag const &properties,
+      winrt::Microsoft::ReactNative::IReactNotificationService const &notifications) noexcept;
+
+  void Destroy() noexcept;
```

Comment:
>Destroy [](start = 7, length = 7)

Can you add quick comment on why should users need this as opposed to just having destructor taking care of this?

Reply:
Added the comment.

---
In reply to: [427623454](https://github.com/microsoft/react-native-windows/pull/4953#discussion_r427623454) [](ancestors = 427623454)